modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -26,6 +26,7 @@ Thus, if you set a retention size limit lower than the maximum size set for the
 If any data blocks exceed the defined retention time or the defined size limit, Prometheus purges these data blocks.
 * If you define a value for `retentionSize` and do not define `retention`, only the `retentionSize` value applies.
 * If you do not define a value for `retentionSize` and only define a value for `retention`, only the `retention` value applies.
+* If you set the `retentionSize` or `retention` value to number 0, the default settings apply. That means, retention time is set to 11 days and retention size is not set.
```

Comment:
```suggestion
* If you set the `retentionSize` or `retention` value to number 0, the default settings apply. That means, retention time is set to 15 days and retention size is not set.
```
there is one bug https://issues.redhat.com/browse/OBSDOCS-884, maybe we can fix it first, then merge this PR

Reply:
Ohh I see, we are just discussing this with @simonpasquier in the comment above. I will get in touch with @bburt-rh about this, thank you!